client/gutenberg/extensions/gif/style.scss

```diff
@@ -1,4 +1,7 @@
 .wp-block-jetpack-gif {
+	clear: both;
+	margin: 0 0 20px;
```

Comment:
We're using `1.5em` as a margin for some other blocks, we might like to match that.

Thoughts @scruffian @thomasguillot ?

Reply:
Could be useful to have SASS variables for things that we want to be consistent across blocks.

Reply:
This came from @simison's suggestion in the CfT, which referenced https://github.com/Automattic/wp-calypso/pull/30436. Did you end up going for ems in Tiled Gallery?

Reply:
Yeah, but I don't think @thomasguillot elaborated more on _why_: https://github.com/Automattic/wp-calypso/pull/30436#issuecomment-458443582

Reply:
Recent a8c themes are using `em` rather than `px` to define the vertical rhythm. `1.5` is a very common number across our themes.